pkg/notes/notes.go

```diff
@@ -473,7 +475,8 @@ func ListCommitsWithNotes(
 		exclusionFilters := []string{
 
 			// 'none' or 'n/a' case insensitive with optional trailing whitespace, wrapped in ``` with/without release-note identifier
-			"(?i)```(release-note\\s*)?(none|n/a)?\\s*```",
+			// the 'none' or 'n/a' can also optionally be wrapped in quotes ' or "
+			"(?i)```(release-note\\s*)?('|\")?(none|n/a)?('|\")?\\s*```",
```

Comment:
Can we adapt the pattern to match this one was well? :)
https://github.com/kubernetes/kubernetes/pull/79892

Reply:
Added via [e628ee3fb6de2f952d20e9ecf4bff5cc80e241d0](https://github.com/kubernetes/release/pull/836/commits/e628ee3fb6de2f952d20e9ecf4bff5cc80e241d0)